server/controller/index.js

```diff
@@ -18,8 +18,21 @@ exports.addQuestion = async (ctx) => {
   return ctx.body;
 };
 
+exports.getAllQuestions = async (ctx) => {
+  await Question.find({})
+    .then((res) => {
+      ctx.body = res;
+      return ctx.body;
+    });
+};
+
 exports.getQuestions = async (ctx) => {
+  const { page } = ctx.request.query;
+  const limit = 10;
```

Comment:
We should also accept a `limit` through `ctx.request.query` with a default value of 10.